### ISSUE TITLE
If there is no  IOU of the anchor and target  exceed the threshold , an error will be reported 

### ISSUE THREAD

**Issue by Programmerwyl** [STVIR/pysot#136]

**Comment by Programmerwyl** [comment]
![2019-07-29 17-06-36屏幕截图](https://user-images.githubusercontent.com/22095854/62035885-4ef83680-b223-11e9-9829-f042afb7a785.png)

**Comment by Programmerwyl** [comment]
1.Firstly,in AnchorTarget/__call__   pos = np.where(overlap > cfg.TRAIN.THR_HIGH)
pos is empty.
then  in loss/select_cross_entropy_loss   pos = label.data.eq(1).nonzero().squeeze().cuda()
pos is empty
next  in loss/get_cls_loss     label = torch.index_select(label, 0, select)
label is empty 
Finally an error is reported in 'F.nll_loss(pred, label)'

**Comment by lb1100** [comment]
It should already have been fixed. Pull the newest code. If the error still exist, we will look into it.

**Comment by Programmerwyl** [comment]
In the latest version, this problem has been resolved.
by this  
 if len(select.size()) == 0 or select.size() == torch.Size([0]):
         return 0
It works
I will close it.
